NineChronicles.Standalone.Executable.sln

```diff
@@ -19,6 +19,16 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Lib9c.Tests", "Lib9c\.Lib9c
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NineChronicles.RPC.Shared", "NineChronicles.RPC.Shared\NineChronicles.RPC.Shared\NineChronicles.RPC.Shared.csproj", "{106573F6-F9C1-41B7-8F02-40E76F177C24}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Lib9c", "Lib9c", "{158143FA-DA22-48D2-A3D1-1AEF0655E257}"
```

Comment:
I think we should remove this line and [below line](https://github.com/planetarium/NineChronicles.Standalone/pull/92/files?file-filters%5B%5D=.sln#diff-ac3c32cd56074f423a59eec8439c1fd2R182) to avoid [MSB5004](https://github.com/planetarium/NineChronicles.Standalone/runs/1241945227#step:4:24)

Reply:
Removed a duplicate Lib9c project.